<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modify</class>
 <widget class="QMainWindow" name="modify">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="usrnameLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/image/blue.png);</string>
    </property>
    <property name="text">
     <string>usrname:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="newpwdLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/image/green.png);</string>
    </property>
    <property name="text">
     <string>newpwd:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="confirmpwdLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/image/green.png);</string>
    </property>
    <property name="text">
     <string>confirmpwd:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="descriptionlabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/image/blue.png);</string>
    </property>
    <property name="text">
     <string>description:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/image/slin.png);
border-image: url(:/new/prefix1/image/blue.png);</string>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usrnameEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="confirmpwdEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="descriptionEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>250</width>
      <height>90</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumodify">
    <property name="title">
     <string>modify</string>
    </property>
   </widget>
   <addaction name="menumodify"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
